compendium/CMakeLists.txt

```diff
@@ -4,6 +4,6 @@ endif()
 add_subdirectory(LogService)
 add_subdirectory(ServiceComponent)
 add_subdirectory(DeclarativeServices)
+add_subdirectory(ConfigurationAdmin)
+add_subdirectory(CM)
```

Comment:
Why "CM" for the subdirectory? That doesn't seem to be in the spirit of the other subdirectories in compendium.

Reply:
"CM" is used here since the osgi spec uses the 'cm' namespace when describing the features introduced by this PR (https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.cm.html#org.osgi.service.cm). The project itself always has the requirement of following the spec as it makes sense for c++ and using "CM" in our implementation is a part of the spec we can reasonably follow in c++ when introducing these changes.

Reply:
Gotcha. Makes sense. Thanks.